src/client.js

```diff
@@ -92,6 +92,20 @@ function createMethod(clientMethod, dbg, cancelCache) {
           call.removeListener('error', onError);
         };
 
+        const originalUnsub = observer.unsubscribe;
+        observer.unsubscribe = function(...args) {
+          // Remove observable errror listener
+          call.removeListener('error', onError);
+          // Add silent error handler. Avoids errors if stream responses have already come in
+          // and we are delaying observable response, like when using delay()
+          call.on('error', () => {});
```

Comment:
Does this take care of the same `domain` error?

Reply:
It takes care of 2 errors I was seeing when calling `unsubscribe` after a delay AND all response streams have already come in. Im not sure there is a way around this because those responseStreams have already run `cb`...

<img width="632" alt="Screen Shot 2019-07-23 at 12 29 07 PM" src="https://user-images.githubusercontent.com/22599518/61741419-af7c0500-ad45-11e9-8c02-52c231897399.png">
<img width="1371" alt="Screen Shot 2019-07-23 at 12 28 48 PM" src="https://user-images.githubusercontent.com/22599518/61741420-af7c0500-ad45-11e9-8667-cbe6ff138316.png">

Reply:
So yea, referencing your comment seems to fix the same domain error. Although your approach seems more robust...